levelkdev/swapr-dapp
# feature: always input max balance after selecting max balance

# Summary

Fixes #857

This creates a way to always show a max balance, even after changing tokens, after clicking on max balance.

<img width="581" alt="image" src="https://user-images.githubusercontent.com/23079677/170029387-78fc0c78-fb76-4f04-87ae-eb3b03898ca5.png">

  # To Test

1. Open the page `swap`
2. Choose an input token with some balance
3. Click max balance
- [ ] Verify it shows the max balance
5. Choose another token with some balance
- [ ] Verify it shows the max balance without clicking max balance button
6. Do the same process for output token
